### ISSUE TITLE
Row component: Babel compilation error

### ISSUE THREAD

**Issue by ShaynaCummings** [pluralsight/design-system#190]
### Expected behavior

No babel compilation error

### Actual behavior

Running into the same compile error with the Row component as was described in https://github.com/pluralsight/design-system/issues/120:

```
ERROR in ../node_modules/@pluralsight/ps-design-system-row/index.js
Module build failed: ReferenceError: Unknown plugin "glamorous-displayname" specified in "/usr/share/smarterer/inquisitor/node_modules/@pluralsight/ps-design-system-row/.babelrc" at 0, attempted to resolve relative to "/usr/share/smarterer/inquisitor/node_modules/@pluralsight/ps-design-system-row"
    at /usr/share/smarterer/inquisitor/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (native)
    at Function.normalisePlugins (/usr/share/smarterer/inquisitor/node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
    at OptionManager.mergeOptions (/usr/share/smarterer/inquisitor/node_modules/babel-core/lib/transformation/file/options/option-manager.js:234:36)
    at OptionManager.init (/usr/share/smarterer/inquisitor/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/usr/share/smarterer/inquisitor/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/usr/share/smarterer/inquisitor/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/usr/share/smarterer/inquisitor/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at transpile (/usr/share/smarterer/inquisitor/node_modules/babel-loader/lib/index.js:46:20)
    at Object.module.exports (/usr/share/smarterer/inquisitor/node_modules/babel-loader/lib/index.js:163:20)
```

If this is the same issue, it looks like `. babelrc` may need to be added to `row/.npmignore` as well.

### Steps to repro

1. `yarn add  @pluralsight/ps-design-system-row`
2. Import the component
3. When running webpack, it will not compile and the above error will show

### Related packages

- row@2.4.20
- core@4.1.4

### Environment

- OS - MacOS 10.13.3
- Browser version - Chrome v64
- Node v6.11.2
- Yarn v0.27.5

**Comment by jaketrent** [comment]
Fixed in row@2.4.21.  

Thanks for filing this, @ShaynaCummings .  Please let me know how it works after this.

**Comment by ShaynaCummings** [comment]
👍 All set, thanks @jaketrent!
